python/py-iminuit/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 name                py-iminuit
 version             1.2
-maintainers         {gmail.com:Deil.Christoph @cdeil} openmaintainer
+maintainers         openmaintainer
```

Comment:
`openmaintainer` by itself is not valid. They should be changed to `nomaintainer`.